src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts

```diff
@@ -111,14 +111,14 @@ export default () => `
 				<div class="resources-container">
 					<h2>${escape(localize('welcomePage.resources', "Resources"))}</h2>
 					<div class="tabs">
-						<input class="input" name="tabs" type="radio" id="tab-1" checked="checked" />
+						<input tabindex="-1" class="input" name="tabs" type="radio" id="tab-1" checked="checked" />
```

Comment:
why are we setting the tab index to -1? this will cause the radio button to be inaccessible

Reply:
I have the checkbox inaccessible for now because it is an unused feature that hasnt been built in yet. The future feature is to allow users to tab between history and pinned files. The checkbox provides no use for the user right now. But the styling and markup are built around what the new feature will be utilizing.

Reply:
i see, I would definitely encourage you to add comment here to state that.